.bazelrc

```diff
@@ -40,6 +40,7 @@ build:clang-tsan --define ENVOY_CONFIG_TSAN=1
 build:clang-tsan --copt -fsanitize=thread
 build:clang-tsan --linkopt -fsanitize=thread
 build:clang-tsan --linkopt -fuse-ld=lld
+build:clang-tsan --linkopt -static-libsan
```

Comment:
Should we add this to `clang-asan` as well (or perhaps even `asan`)?

Reply:
asan does dynamic link to save disk space so it doesn't need this. (static_test is disabled too)